record-and-playback/deploy.sh

```diff
@@ -33,14 +34,23 @@ function deploy_format() {
 	for format in $formats
 	do
 		playback_dir="$format/playback/$format"
+		if [ $format == "screenshare" ]; then
+			playback_dir="$format/playback"
+		fi
 		scripts_dir="$format/scripts"
-		if [ -d $playback_dir ]; then sudo cp -r $playback_dir /var/bigbluebutton/playback/; fi
+		nginx_file="$format/scripts/*.nginx"
+		if [ -d $playback_dir ]; then
+			if [ "$format" == "presentation" ]; then sudo cp -r $playback_dir /var/bigbluebutton/playback/; fi
+			if [ "$format" == "screenshare" ]; then sudo mkdir -p /usr/local/bigbluebutton/core/playback/$format; sudo cp -r $playback_dir/* /usr/local/bigbluebutton/core/playback/screenshare/; fi
+			if ([ "$format" != "presentation" ] & [ "$format" != "screenshare" ]); then sudo mkdir -p /usr/local/bigbluebutton/core/playback/$format; sudo cp -r $playback_dir /usr/local/bigbluebutton/core/playback/; fi
+		fi
 		if [ -d $scripts_dir ]; then sudo cp -r $scripts_dir/* /usr/local/bigbluebutton/core/scripts/; fi
+		if [ -f $nginx_file ]; then sudo cp $scripts_dir/*.nginx /usr/share/bigbluebutton/nginx/; fi
 		sudo mkdir -p /var/log/bigbluebutton/$format /var/bigbluebutton/published/$format /var/bigbluebutton/recording/publish/$format
 	done
 }
 
-deploy_format "presentation"
+deploy_format "presentation screenshare notes"
```

Comment:
Actually in the screenshot in the PR description it says this will remain as `presentation` only and people have to add manually the newly requested formats...
```suggestion
deploy_format "presentation"
```